app/components/viral/alert_component.rb

```diff
@@ -33,9 +33,9 @@ class AlertComponent < Viral::Component
     # Maps incoming logical types (including Rails flash keys) to canonical UI states
     TYPE_MAPPINGS = {
```

Comment:
suggestion (blocking): Why not update all the types to be symbols instead of mixed with strings.